app/assets/javascripts/passengers/passengers.js

```diff
@@ -36,6 +36,7 @@ $( document ).on("turbolinks:load", function() {
 
   $('#passenger_eligibility_verification_attributes_verifying_agency_id').change(function(){
     var needsContactInfo = $(this).children("option:selected").data('needs-contact-info');
+    needsContactInfo ||= false; // blank option will needlessly cause toggling
```

Comment:
Chromium version 85 or newer is required to support the `||=` operator. I recommend you set this without this new fangled operator for comparability's sake. 
```suggestion
    needsContactInfo = needsContactInfo || false; // blank option will needlessly cause toggling
```

Reply:
`||=` seems like such a basic operation that I'd have _never_ guessed it would be unsupported in even older browsers. Thanks for debugging this for me! <3